test/github-issues/10734/entity/Company.ts

```diff
@@ -0,0 +1,23 @@
+import {
+    Entity,
+    OneToMany,
+    PrimaryColumn,
+    VirtualColumn,
+    BaseEntity,
+} from "../../../../src"
+import Employee from "./Employee"
+
+@Entity({ name: "companies" })
+export default class Company extends BaseEntity {
+    @PrimaryColumn("varchar", { length: 50 })
+    name: string
+
+    @VirtualColumn({
+        query: (alias, context) =>
+            `SELECT COUNT("name") FROM "employees" WHERE "companyName" = ${alias}.name AND age > ${context.minimumAge}`,
```

Comment:
Actually consistent quotations are more important in the code, especially if we want to expand the test to other DB drivers.